<launch>
  <arg name="vehicle_control_topic" default="/vehicle_control" />
  <arg name="ego_state_topic" default="/ego_state_feedback" />

  <node pkg="vehicle" type="hospital_bed_node.py" name="hospital_bed_node" output="screen">
    <param name="control_subscriber_topic" value="$(arg vehicle_control_topic )" />
    <param name="state_publisher_topic" value="$(arg ego_state_topic)" />
  </node>

</launch>